src/octopeer-github/content/ContentController.ts

```diff
@@ -105,7 +105,10 @@ class ContentController {
         this.unhookFromDOM();
         this.hookSemanticToDOM(database);
         this.hookTrackersToDOM(database);
-        this.hookMutationObserverToDOM();
+
+        if (Options.get("dataHTML")) {
```

Comment:
Would be better, or else please consider making "dataHTML" a public static final (or whatever TS equivalent you can do)